### PR TITLE
698: Fix changeEnrollment for enrollment with funding

### LINKED ISSUE #698: Cannot save new enrollments when adding a new record or editing an existing record
Screen Shot 2020-11-09 at 3 32 06 PM

### PR DESCRIPTION
**Pull request by jlhogan**
closes #698

### CODE REVIEW

**Review by bamader — APPROVED** [review]
Love these easy fixes--looks good!

File: client/src/containers/EditRecord/EnrollmentFunding/ChangeEnrollment/Form.tsx [hunk 1]
```diff
@@ -109,7 +109,6 @@ export const ChangeEnrollmentForm: React.FC<ChangeEnrollmentFormProps> = ({
             optional={true}
             label={`Last reporting period for current ${activeFunding.fundingSpace?.source} - ${activeFunding.fundingSpace?.time} funding`}
           />
-          )
```
Comment (mmazanec22) [inline client/src/containers/EditRecord/EnrollmentFunding/ChangeEnrollment/Form.tsx:112]: ?

**Review by mmazanec22 — APPROVED** [review]
Approved but also maybe did you not mean to remove that parenthesis?

**Review by JordanGuinn — APPROVED** [review]

File: client/src/containers/EditRecord/EnrollmentFunding/ChangeEnrollment/Form.tsx (hunk 1 above)
Comment (jlhogan) [reply client/src/containers/EditRecord/EnrollmentFunding/ChangeEnrollment/Form.tsx:112]: this is within the JXS, was getting rendered on the page

**Pull request merged**